bin/migrate-mongo.js

```diff
@@ -58,7 +58,8 @@ program
 program
   .command("up")
   .description("run all pending database migrations")
-  .option("-f --file <file>", "use a custom config file")
+  .option("-c --custom <custom>", "use a custom config file")
```

Comment:
The description of this option is the same as the above, maybe it was a mistake when copying the line above to add a new option...

Reply:
A typo. Fixed.